### DISCUSSION TITLE
Animate fragment or transition multiple slides, to achieve sliding content but keep one part static (for a few slides, so not header/footer)

### DISCUSSION THREAD

**Discussion by hipeilin** [hakimel/reveal.js#3388] [Q&A] [answered]
Let's consider these 3 slides of the presentation (which contain a lot more slides), they share the same heading, let's say an h1 "Day 1", and the content contains h2 "Morning", "Noon", and "Evening". The result I try to achieve is to keep the heading "Day 1" static and have the h2 "Morning" slide to the left, simultaneously have the h2 "Noon" slide in from the right and land at the same position. Is this possible? I tried having different `section`s but the whole thing (including the heading h1) will be slided. I saw there is also `fragment`s, but I can only find fade animation. Any tips?

**Comment by Martinomagnifico** [comment, marked as answer]
Auto-animate might work here.

**Reply by hipeilin** [reply]
Alright, I will give it a try in the future if I use it again. For the time being, I just used PowerPoint and its Morph feature to achieve that effect.
